px-app-nav.html

```diff
@@ -247,6 +247,8 @@
       //this._checkNavExpandedState();
       // this._checkNavExpanded();
```

Comment:
If this code is dead, we can remove it. Commented out code just confuses people :)